Lab1/index.html

```diff
@@ -0,0 +1,126 @@
+                        <li class="header__nav-item"><a href="">Services</a></li>
+                        <li class="header__nav-item"><a href="">Enterprise</a></li>
+                        <li class="header__nav-item"><a href="">Pricing</a></li>
+                    </ul>
+                </nav>
+                <button class="header__try">Try it for free</button>
+            </header>
+        </div>
+        <div class="container">
+            <div class="container__hero">
+                <div class="hero__inner">
+                    <h1 class="title">Take your relationships to the next level</h1>
+                    <p class="paragraph">The multichannel sales engagement platform which automates personal email outreach, calls, and tasks, while Sales can focus on what really matters - closing deals.</p>
+                    <div class="email-to-try">
+                        <input type="email" style="width: 330px; height: 41.58px;"; class="input-email" value="Email id"
+                        onblur="if(this.value==''){ this.value='Email id'; this.style.color='#73738D99';}"
+                        onfocus="if(this.value=='Email id'){ this.value=''; this.style.color='#000';}"
+                        style="color:#BBB;" />
+                        <button class="submit-email">Try it for free</button>
+                    </div>
+                </div>
+                <div class="phone">
+                <img src="./images/PhoneCircleTop.png" alt="" class="phone-circle-top">
+                <img src="./images/PhoneCircleBottom.png" alt="" class="phone-circle-bottom">
+                <img src="./images/phone.png" alt="" class="phone-img">
+                </div>
+            </div>
+        </div>
+   </section>
+   <section class="info">
+        <div class="container">
+            <div class="notifications">
+                <div class="notification"><img src="./images/img1.png" class="user-img"><p class="notification-text">You have a meeting with <b>Surbhi</b> to help her with her portfolio. She got a new pet (Dog named Tiger).</p></div>
+                <div class="notification"><img src="./images/img2.png" class="user-img"><p class="notification-text"><b>Ripal</b> has recently been to Thailand and she wants to discuss about new Job opportunity.</p></div>
+                <div class="notification"><img src="./images/img3.png" class="user-img"><p class="notification-text"><b>Prashasti</b> hasn’t responded to your Email from 22 Jan, 2020 <b>Want to check up on her? <h8 style="color:MediumSeaGreen;">Follow up</h8></b></p></div>
+            </div>
+            <div class="reminders"> 
+                <h1 class="title-1">Get reminders to follow up</h1>
+                <p class="paragraph-reminder">We remind you of everything important and track everything that’s important to your important ones so you always stay connected. We only tell you the details when you need them the most.</p>
+                <ul class="details-1">
```

Comment:
Please change to h6  or another (in html they are from h1 to h6 (but h1 adjust to use once for the whole page)) ;)